osu.Game/Graphics/Containers/Markdown/Footnotes/OsuMarkdownFootnoteTooltip.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Markdig.Extensions.Footnotes;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Containers.Markdown;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Overlays;
+using osuTK;
+
+namespace osu.Game.Graphics.Containers.Markdown.Footnotes
+{
+    public partial class OsuMarkdownFootnoteTooltip : CompositeDrawable, ITooltip
+    {
+        private readonly FootnoteMarkdownContainer markdownContainer;
+
+        [Cached]
+        private OverlayColourProvider colourProvider;
+
+        public OsuMarkdownFootnoteTooltip(OverlayColourProvider colourProvider)
+        {
+            this.colourProvider = colourProvider;
+
+            Masking = true;
+            Width = 200;
```

Comment:
This doesn't match web, but web has constraints and requirements that are way too complicated to make happen here (min width 50px, max width 280px, content vertically centred). This is intended to be best-effort.

Reply:
Pretty unfortunate with this one, but I guess it's better than nothing for now.